ExampleRDoc.rdoc

```diff
@@ -1,3 +1,5 @@
+= Example RDoc
```

Comment:
This needs to be escaped or it'd link to the RDoc constant

```suggestion
= Example \RDoc
```

(I'm aware that this is not intuitive but changing it would be a huge breaking change to some users)